### PR TITLE
readme: update link to tarantool manual

### LINKED ISSUE #130: readme: outdated link to tarantool manual
In section "Hello World" -> "Connectors" URL is outdated and returns 404.

### PR DESCRIPTION
**Pull request by 0x501D**
closes #130

### CODE REVIEW

**Review by LeonidVas — APPROVED** [review]
As for me,"update" is more correct than "fix". Up to you.
LGTM.

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ the `import {...}` section at the start of any Go program.
 
 <h2>Hello World</h2>
 
-In the "[Connectors](http://tarantool.org/doc/book/connectors/index.html#go)"
+In the "[Connectors](https://www.tarantool.io/en/doc/latest/book/connectors/#index-connector-go)"
 chapter of the Tarantool manual, there is an explanation of a very short (18-line)
 program written in Go. Follow the instructions at the start of the "Connectors"
 chapter carefully. Then cut and paste the example into a file named `example.go`,
```
Comment (Totktonada) [inline README.md:67]: The paragraph states that the link contains an example of connecting from Go. I guess it is https://www.tarantool.io/en/doc/latest/getting_started/getting_started_go/ so.

Comment (0x501D) [reply README.md]: fixed

**Review by Totktonada — APPROVED** [review]
LGTM.

**Comment by Totktonada** [comment]
I think that's trivial enough to don't ask review from Yura and just proceed.

**Pull request merged**